pkgs/development/tools/haskell/haskell-language-server/hls-ghcide.nix

```diff
@@ -0,0 +1,68 @@
+# A forked version of ghcide by the haskell-language-server team
+#
+# This file was generated using cabal2nix and required manual edition,
+# there is no automated way to upgrade it.
+# 1. upgrade the git revision
+# 2. trial and error to make it work
```

Comment:
Like I said above, the automated way to upgrade this should be to just re-run `cabal2nix`.

Any sort of trail and error to make it work (like adding the git dependency, etc) should be added in the `configuration-common.nix` file, not manually in these two files.